src/fu-usb-device.c

```diff
@@ -372,6 +372,30 @@ fu_usb_device_get_platform_id (FuUsbDevice *self)
 	return g_usb_device_get_platform_id (priv->usb_device);
 }
 
+/**
+ * fu_usb_device_get_spec:
+ * @self: A #FuUsbDevice
+ *
+ * Gets the string USB revision for the device.
+ *
+ * Return value: a specification revision in BCD format, or 0x0 if not supported
+ *
+ * Since: 1.3.4
+ **/
+guint16
+fu_usb_device_get_spec (FuUsbDevice *self)
+{
+#if G_USB_CHECK_VERSION(0,3,1)
```

Comment:
I think it might be good for our future selves to leave a comment here that custom flags used in `fu-vli-usbhub-device.c` go away when bumping this version.

Reply:
I'm hoping to do a new release soonish and raise the fwupd dep.